examples/dummy_plugin/dummy_plugin/api/views.py

```diff
@@ -9,3 +13,21 @@ class DummyViewSet(ModelViewSet):
     queryset = DummyModel.objects.all()
     serializer_class = DummyModelSerializer
     filterset_class = DummyModelFilterSet
+
+
+#
+# Webhook Testing
+#
+
+
+class DummyModelWebhook(APIView):
+    """
+    Dummy view used in testing webhooks for plugins.
+    """
+
+    permission_classes = [IsAuthenticated]
```

Comment:
Do you see a benefit to requiring authentication for this test API endpoint? It makes the test a bit more complicated to implement since you have to construct and include a user Token as part of the test, which doesn't to me seem directly related to the purpose of the test case to exercise Webhook functionality?